### PR TITLE
[Stablehlo] Add stablehlo support for aten.abs

### PR DESCRIPTION
**Pull request by AmosLewis**

### CODE REVIEW

File: test/Conversion/TorchToStablehlo/elementwise.mlir [hunk 1]
```diff
@@ -624,3 +624,16 @@ func.func @torch.aten.div.Tensor_mode$floor(%arg0: !torch.vtensor<[?,?,?,?],f32>
   return %0 : !torch.vtensor<[?,?,?,?],f32>
 }
 
+// -----
```
Comment (qingyunqu) [inline test/Conversion/TorchToStablehlo/elementwise.mlir:627]: I think you should write a e2e test, instead of lit test.

**Review by qingyunqu — CHANGES_REQUESTED** [review]

File: test/Conversion/TorchToStablehlo/elementwise.mlir (hunk 1 above)
Comment (qingyunqu) [reply test/Conversion/TorchToStablehlo/elementwise.mlir:627]: Do you konw why "ElementwiseAbsModule_basic" is still failing on stablehlo after adding `aten.abs` support?

Comment (AmosLewis) [reply test/Conversion/TorchToStablehlo/elementwise.mlir:627]: For stablehlo, it passes. As you can see here:
`python -m e2e_testing.main -c stablehlo -f "ElementwiseAbsModule_basic" -v `
```
2023-05-07 20:54:50.204679: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-05-07 20:54:50.306760: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2023-05-07 20:54:50.306781: I tensorflow/compiler/xla/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2023-05-07 20:54:50.852281: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
2023-05-07 20:54:50.852335: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
2023-05-07 20:54:50.852342: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
Compiling ElementwiseAbsModule_basic...
Running ElementwiseAbsModule_basic...
PASS - "ElementwiseAbsModule_basic"

Summary:
    Passed: 1
```
Where did you see it fail?

Comment (qingyunqu) [reply test/Conversion/TorchToStablehlo/elementwise.mlir:627]: ok，sorry for my mistake，LGTM

**Review by qingyunqu — APPROVED** [review]

**Review by ZihengJiang — APPROVED** [review]

**Pull request merged**